src/util/animation/AnimationBase.ts

```diff
@@ -17,42 +17,66 @@ const defaultAbort = () => false;
 export abstract class AnimationBase<
   T extends number | number[] = number | number[]
 > {
+  /**
+   * @see TAnimationBaseOptions
```

Comment:
why this?

Reply:
It improves clarity IMO, but if you think it's too verbose it's not strictly necessary

Reply:
Is it visible anywhere for the dev accept here?
If so I don't see how helpful it is.
If the dev is here they can see the typedefs themselves and use VSCODE to navigate to definitions.

Reply:
That's true, but I just like having extra pointers that make stuff more obvious. I'll keep it unless you specifically want it removed.

Reply:
So it is extra code without a gain
I prefer removed

Reply:
actually I am ok with this.
I will push the fix including this